docker/Dockerfile.main

```diff
@@ -11,9 +11,12 @@ FROM ${REGISTRY}/hypergiant-jupyter:cpu-base AS cpu-firefly
 ADD docker/entrypoint-torch.sh /usr/local/bin/
 ADD docker/entrypoint-local.sh /usr/local/bin/
 ADD ./hypertrain /home/jovyan/hypertrain
+ADD ./executor /home/jovyan/executor
+ADD ./executor /tmp/repo
```

Comment:
Why add it in two places?

Reply:
Good question.

The `ADD` on line 14 is for `pip install`-ing the package. After it is installed, we'll delete the `executor` (see lines 28-32 of `Dockerfile.main`).

The `ADD` on line 15 is for the `entrypoint-local.sh`, when we copy the demo notebooks, sample data, and yaml files. Please see lines 10-16 of `entrypoint-local.sh`.